requirements.txt

```diff
@@ -8,4 +8,9 @@ pandas
 tqdm
 pypet
 deap
-dill
\ No newline at end of file
+dill
+symengine
+sympy
+git+https://github.com/neurophysik/jitcdde.git#egg=jitcdde
```

Comment:
newest `jitcdde` package for adaptive `dt` backend

Reply:
You think we can ask the guy to put it on pip?

Reply:
sure, I'll ask him

Reply:
https://github.com/neurophysik/jitcdde/issues/29